src/index.js

```diff
@@ -175,6 +176,7 @@ exports.init = (Implementation) => {
   }
 
   new SessionRoute(app, opts).perform();
+  new HealthCheckRoute(app, opts).perform();
```

Comment:
If possible, I would reorder them, for something more natural/logical 
```
new HealthCheckRoute(app, opts).perform();
new SessionRoute(app, opts).perform();
```
healthcheck is a very basic route without logic (basic public route).
Session routes are made for a more complex usage (public route to authenticate)